dev/tests/integration/testsuite/Magento/Swatches/_files/swatch_attribute_color.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+use Magento\Eav\Api\Data\AttributeOptionInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+
+/** @var \Magento\Framework\ObjectManagerInterface $objectManager */
+$objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
```

Comment:
Why do we need this fixture if we already have `dev/tests/integration/testsuite/Magento/Swatches/_files/swatch_attribute.php`?